### PR TITLE
Project sign up form (copy from google forms)

### PR DESCRIPTION
**Pull request by Mariatta**
I think it is easier to comment on GitHub.
I will update the form based on feedback on this PR.

### CODE REVIEW

File: forms/project-signup.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Google form link: https://docs.google.com/forms/d/e/1FAIpQLSd3D0BvnhgLRbtK2LOC-aAAkvZt484RCpwY0lpFdsFljEy9ig/viewform?usp=sf_link
+
+# Mentored Sprint at PyCon US: Project Sign Up
+
+Please fill in this form if you manage an open source project, and can commit to provide four hours 
+of mentorship to a diverse group of contributors during the sprint.
```
Comment (trallard) [inline forms/project-signup.md]: and can commit to providing four or two hours  (matching the selection below)

**Comment by Nikoleta-v3** [comment]
No comments on this. It looks really good :+1:

**Comment by nnja** [comment]
@Mariatta, I left some feedback in the slack channel but I'll repost here for the record. It looks like two out of three have been resolved.

- [X] make project areas a multi-select checkbox
- [X] Add description to availability during the sprint
- [ ] Make mentor names and email addresses a larger field

Is the third one even possible? I'm not sure if Google Forms supports larger text areas.

**Comment by nnja** [comment]
One more question -- is the form and resulting data shared with and able to be edited by all the organizers?

**Comment by Mariatta** [comment]
Thanks @nnja!

> Make mentor names and email addresses a larger field

The field has been changed from "short answer text" to "paragraph", unfortunately Google forms still shows this as a single line initially ☹️  But you can press enter and type in multiple lines.. I don't know how if there's a way to render this differently.

> One more question -- is the form and resulting data shared with and able to be edited by all the organizers?

Yes, once we're happy with the form and can start accepting submissions, I'll enable the google sheets recording (so any form submission goes to google sheets directly) I'll make sure to share the spreadsheets with us all.

**Comment by nnja** [comment]
@Mariatta ah, I see. Maybe expand the message to: "Press enter to expand the text field and add new lines." - What do you think?

Besides for that small nit and this comment https://github.com/pycon-mentored-sprints/organisational/pull/3#discussion_r253775927 LGTM.

**Comment by Mariatta** [comment]
Thanks! I've updated the forms, responses will be automatically recorded in a Google Sheets.
I believe you should also receive notification that you have access to both the form and the Google Sheets.

If you like I can also make it so we all receive email updates whenever there is a new submission.

**Comment by Mariatta** [comment]
If you don't receive access to the forms/google sheets let me know I'll try to invite you again.

**Pull request merged**